sense platform/sensors/CSScreenSensor.m

```diff
@@ -116,29 +121,39 @@ - (void) dealloc {
 	self.isEnabled = NO;
 }
 
+
+// Whenever no lockcomplete event was received, we assume the screen has been turned on
+- (void) lockcompleteNotReceived {
+	[refToSelf commitDisplayState:YES];
```

Comment:
@Tubyhes yeah because of those darwin notifications are in a static scope, you don't actually have a self instance; therefore this... came from Pim btw.